src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -230,6 +231,9 @@ public class KNNSettings {
         NodeScope,
         Dynamic
     );
+
+    public static final Setting<Boolean> KNN_FAISS_AVX2_DISABLED_SETTING = Setting.boolSetting(KNN_FAISS_AVX2_DISABLED, false, NodeScope);
```

Comment:
Is this a dynamic setting? and how a customer can set this setting?

Reply:
no, this is not dynamic. Right now, users can set this from opensearch.yml

Reply:
got it.